disnake/state.py

```diff
@@ -1437,6 +1438,73 @@ def parse_guild_role_update(self, data) -> None:
                 data["guild_id"],
             )
 
+    def parse_guild_scheduled_event_create(self, data) -> None:
+        scheduled_event = GuildScheduledEvent(state=self, data=data)
+        guild = scheduled_event.guild
+        if guild is not None:
+            guild._scheduled_events[scheduled_event.id] = scheduled_event
+        self.dispatch("guild_scheduled_event_create", scheduled_event)
+
+    def parse_guild_scheduled_event_update(self, data) -> None:
+        guild = self._get_guild(int(data["guild_id"]))
+
+        if guild is None:
+            _log.debug(
+                "GUILD_SCHEDULED_EVENT_UPDATE referencing unknown guild ID: %s. Discarding.",
+                data["guild_id"],
+            )
+            return None
```

Comment:
```suggestion
            return
```

[same suggestion below]